src/app/stacks/multivisionstack.cpp

```diff
@@ -27,6 +27,9 @@ MultiVisionStack::MultiVisionStack(string _name, RenderOptions * _opts) {
 }
 
 MultiVisionStack::~MultiVisionStack() {
+  for (int i=0;i<threads.size();i++) {
```

Comment:
It would be safest to call stop() and then wait() on each thread before deleting them.